src/components/DataTable/index.jsx

```diff
@@ -159,48 +159,53 @@ const DataTable = ({
   }, [filters]);
 
   return (
-    <div {...getTableProps()} className={className}>
+    <div {...getTableProps()} className={className} tabIndex="0">
       <div className={tableContainerClassName}>
-        {headerGroups.map((headerGroup) => (
-          <div
-            {...headerGroup.getHeaderGroupProps()}
-            className={headerGroupClassName}
-          >
-            {headerGroup.headers.map((column) => (
-              <>
-                <div
-                  className={headerCellClassName}
-                  {...column.getHeaderProps(
-                    columnSort ? column.getSortByToggleProps() : undefined
-                  )}
-                >
-                  <span className={headerCellTextClassName}>
-                    {column.render("Header")}
-                  </span>
-                  {columnSort && (
-                    <span
-                      className={
-                        column.isSorted
-                          ? column.isSortedDesc
-                            ? columnIsSortedDecClassName
-                            : columnIsSortedAscClassName
-                          : columnIsSortedClassName
-                      }
-                    />
-                  )}
-                </div>
-                {columnResize && (
-                  <div
-                    {...column.getResizerProps()}
-                    className={`${tableColumnResizer} ${
-                      column.isResizing ? tableColumnIsResizing : ""
-                    }`}
-                  />
-                )}
-              </>
-            ))}
-          </div>
-        ))}
+        {headerGroups.map((headerGroup) => {
+          const { key } = headerGroup.getHeaderGroupProps()
```

Comment:
Added a key here and on 167 to fix React warning/errors of list items not having unique keys.